src/requirements.yml

```diff
@@ -3,8 +3,6 @@
   name: amazon_ssm_agent
 - src: https://github.com/cisagov/ansible-role-automated-security-updates
   name: automated_security_updates
-- src: https://github.com/cisagov/ansible-role-backports
```

Comment:
We should probably also remove the following that relied on this dependency:
https://github.com/cisagov/openvpn-packer/blob/68e28fc5edddc9f85343fa8cdebf989b9e39bc7e/src/aws.yml#L8-L9
https://github.com/cisagov/openvpn-packer/blob/68e28fc5edddc9f85343fa8cdebf989b9e39bc7e/src/aws.yml#L15-L28

Reply:
All of that stuff is [already](https://github.com/cisagov/openvpn-packer/blob/lineage/skeleton/src/requirements.yml) [gone](https://github.com/cisagov/openvpn-packer/blob/lineage/skeleton/src/aws.yml), no?

Reply:
Well that's peculiar I wasn't seeing it in my review page. Apologies and carry on!